src/Strings.php

```diff
@@ -0,0 +1,28 @@
+<?php namespace Lean\Utils;
+
+/**
+ * A suite of functions for working strings.
+ *
+ * Class Strings.
+ *
+ * @package Lean\Utils
+ */
+class Strings
```

Comment:
They're not in PHP7, which will cause issues in the future. PHPStorm gives me this warning: http://snaps.getmoxied.net/1R071d3h2K44